src/docs/cookbook/animation/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Animation
+title: Hoạt ảnh
```

Comment:
Should we keep "animation" the same? Regardless of the decision, remember to put this to glossary.

Reply:
Yeah i also don't know any word in vietnamese that suitable for Animation so i think we should keep "animation" the same